packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -75,6 +85,15 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
     /* @conditional-compile-remove(call-readiness) */ onNetworkingTroubleShootingClick
   } = props;
 
+  /* @conditional-compile-remove(video-background-effects) */
+  const [showVideoEffectsPane, setVideoEffectsPane] = useState(false);
+  /* @conditional-compile-remove(video-background-effects) */
+  const setShowVideoEffectsPane = useCallback(
```

Comment:
nit: I don't think you need this callback. You already have the useState setter.

Reply:
done